include/openssl/store.h

```diff
@@ -272,6 +276,10 @@ typedef OSSL_STORE_INFO *(*OSSL_STORE_load_fn)(OSSL_STORE_LOADER_CTX *ctx,
                                                void *ui_data);
 int OSSL_STORE_LOADER_set_load(OSSL_STORE_LOADER *loader,
                                OSSL_STORE_load_fn load_function);
+typedef OSSL_STORE_INFO *(*OSSL_STORE_load_with_libctx_fn)
+    (OSSL_STORE_LOADER_CTX *ctx, const UI_METHOD *ui_method, void *ui_data,
+     OPENSSL_CTX *libctx, const char *propq);
```

Comment:
[sigh] there's a pattern throughout OSSL_STORE, where the UI stuff comes last, and diverse callbacks come even more last...

So here's a choice, change this signature in this PR, or leave it to me to do so in #12512.  I would prefer the former, but this PR has been waiting for a while, so...

Reply:
another PR would be a good idea :)

Reply:
If you choose to change it, here's what I'd prefer to see:

``` C
typedef OSSL_STORE_INFO *(*OSSL_STORE_load_with_libctx_fn)
    (OSSL_STORE_LOADER_CTX *ctx, OPENSSL_CTX *libctx, const char *propq,
     const UI_METHOD *ui_method, void *ui_data);
```

Reply:
> another PR would be a good idea :)

Hahaha, I think that making a whole new PR just to re-arrange this parameters is a bit over the top.

Reply:
Ahh ok this is a leftover in the header - load_with_libctx_fn is not needed now. Will remove.

Reply:
Removed